stdlib/camlinternalFormat.ml

```diff
@@ -1952,6 +1952,33 @@ let rec strput_acc b acc = match acc with
   | Acc_invalid_arg (p, msg) -> strput_acc b p; invalid_arg msg;
   | End_of_acc               -> ()
 
+(******************************************************************************)
+                      (* Heterogeneous printf variants *)
+
+module Arg_list = struct
+  (** List of heterogeneous values.
```

Comment:
This may be a detail, but generally, documentation is placed in the `mli`, so I suggest using `(*` as a comment opening, to act as a "guide to an implementation detail" rather than actual documentation.

Reply:
That makes sense!